src/libraries/Microsoft.PowerFx.Interpreter/ReflectionFunction.cs

```diff
@@ -252,6 +252,12 @@ public FormulaValue Invoke(FormulaValue[] args)
     // A function capable of async invokes. 
     internal interface IAsyncTexlFunction
     {
+        DPath Namespace { get; }
+
+        string Name { get; }
+
+        DType ReturnType { get; }
```

Comment:
why are we adding this to the interface?

Reply:
you're right, this isn't necessary